dbus-wait.sh

```diff
@@ -6,7 +6,8 @@ function wait_for_dbus() {
 			  /org/freedesktop/DBus \
 			  org.freedesktop.DBus.ListNames
 
-		if [ $? ]; then
+                dbus_wait=$?
```

Comment:
@shawaj Minor indentation inconsistency here: the file is indented with tabs, this doesn't line up.